index.html

```diff
@@ -317,6 +317,25 @@ <h2 class="connect-link">LET'S CONNECT</h2>
           </li>
         </ul>
       </section>
+      <section class="fotter-contact-form-container">
+        <section class="fotter-contact-form-text">
+          <h2 class="fotter-contact-form-heading">Contact me</h2>
+          <p class="fotter-contact-form-subheading">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+            I’d love to help with it!</p>
+             <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] Hey there! I noticed that the form_id placeholder in the form's action URL is not set up properly. To fix this, you'll need to sign up for [Formspree](https://formspree.io/) and create a form to get a proper form ID. Once you have the ID, please replace the form_id placeholder in the form's action URL with the ID. Let me know if you have any questions or need further assistance. Thank you!